File: Java/Lista 4/L4_Ex1.1/src/l4_ex1/pkg1/Livro.java

package l4_ex1.pkg1;

public class Livro {
    private String nome;
    private String autor;
    private String anoPublicacao;

    public Livro(String nome, String autor, String anoPublicacao) {
        this.nome = nome;
        this.autor = autor;
        this.anoPublicacao = anoPublicacao;
    }

    public String getNome() {
        return nome;
    }

    public String getAutor() {
        return autor;
    }

    public String getAnoPublicacao() {
        return anoPublicacao;
    }
}

File: Swift/L4/Lista_04_EX02/Livro.swift
public class Livro {
    private var nome: String
    private var autor: String
    private var anoPublicacao: String

    init(nome: String, autor: String, anoPublicacao: String) {
        self.nome = nome
        self.autor = autor
        self.anoPublicacao = anoPublicacao
    }

    func getNome() -> String {
        return nome
    }

    func getAutor() -> String  {
        return autor
    }

    func getAnoPublicacao() -> String {
        return anoPublicacao
    }
}